Content.Server/Medical/CrewMonitoring/CrewMonitoringConsoleSystem.cs

```diff
@@ -163,56 +166,42 @@ private bool HasCorpsesOutsideMorgue(CrewMonitoringConsoleComponent component)
     /// </summary>
     private bool IsEntityInMorgue(EntityUid entity)
     {
-        // Check if the entity is contained within any morgue
-        var morgueQuery = EntityQueryEnumerator<MorgueComponent, EntityStorageComponent>();
+        var parent = Transform(entity).ParentUid;
 
-        while (morgueQuery.MoveNext(out var morgueUid, out var morgue, out var storage))
-        {
-            // Check if the entity is contained in this morgue
-            if (storage.Contents.ContainedEntities.Contains(entity))
-            {
-                return true;
-            }
-        }
-
-        return false;
+        return parent.IsValid() && HasComp<MorgueComponent>(parent);
```

Comment:
```suggestion
        return HasComp<MorgueComponent>(parent);
```

В этом `parent.IsValid()` не было смысла